examples/12. Advanced Action.md

```diff
@@ -0,0 +1,66 @@
+FORMAT: 1A
+
+# Advanced Action API
+An action in a resource in the API Blueprint can be considered as a state transition. This API example demonstrates that a resource can contain a state transition to another resource. This allows us to show that the resource `Task` can afford an action that is not limited to the resource `Task`.
```

Comment:
> An action in a resource in the API Blueprint can be considered as a state transition

A resource action is – in fact – a state transition. 

> This API example demonstrates that a resource can contain a state transition to another resource.

This API example demonstrates an action – state transition – to another resource. 

>   This allows us to show that the resource `Task` can afford an action that is not limited to the resource `Task`.

I would remove this sentence completely